src/ApplicationManager.cpp

```
@@ -1440,3 +1446,50 @@ bool ApplicationManager::sendAssignSpool(const char* toolNumber) {
     return true;  // Native test: pretend success
 #endif
 }
+
+bool ApplicationManager::publishPendingSpool(int spoolmanId) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)